pkg/parcacol/parcacol.go

```diff
@@ -59,17 +60,41 @@ func FlatProfileToBuffer(logger log.Logger, ls labels.Labels, schema *dynparquet
 
 	rows := make(Samples, 0, len(prof.FlatSamples))
 	for _, s := range prof.FlatSamples {
+		pprofLabels := labels.Labels{}
+		for name, values := range s.Label {
+			if len(values) > 1 {
+				panic("don't expect more than 1 value per pprof label")
+			}
+			pprofLabels = append(pprofLabels, labels.Label{
+				Name:  name,
+				Value: values[0],
+			})
+		}
+
+		pprofNumLabels := labels.Labels{}
+		for name, values := range s.NumLabel {
+			if len(values) > 1 {
+				panic("don't expect more than 1 value per pprof num label")
+			}
+			pprofNumLabels = append(pprofNumLabels, labels.Label{
+				Name:  name,
+				Value: strconv.FormatInt(values[0], 10),
```

Comment:
We can have dynamic columns of type int, why not use that?

Reply:
Yes, now that I understand how the dynamic columns work, I agree. Was just easier, haha. Will update this!